src/components/modals/TransactionAdvanceDetails.js

```diff
@@ -40,27 +39,15 @@ export class TransactionAdvanceDetailsComponent extends PureComponent<Props> {
   openUrl = () => {
     const { url } = this.props
     if (url == null || url === '') return
-    openBrowserUri(url)
+    openBrowserUri({ uri: url, isSafariView: true })
```

Comment:
do not use a Safari webview

Reply:
This code originally opened SafariView, if available. Was the comment a request to change existing behavior?

[`openBrowserUri`](https://github.com/EdgeApp/edge-react-gui/blob/dacab0259743bb540e4782346f40becb3bf261eb/src/util/WebUtils.js#L14) has a parameter `isSafariView ` to try to use SafariView, only if available.

Reply:
This code originally opened SafariView, if available. Was the comment a request to change existing behavior?

[openBrowserUri](https://github.com/EdgeApp/edge-react-gui/blob/dacab0259743bb540e4782346f40becb3bf261eb/src/util/WebUtils.js#L14) has a parameter isSafariView to try to use SafariView, only if available.